src/prommis/uky/costing/ree_plant_capcost.py

```diff
@@ -797,12 +841,12 @@ def total_plant_cost_rule(self):
                                 + self.total_variable_OM_cost[0]
                             )
                             / (
-                                self.recovery_rate
+                                self.recovery_rate_per_year
                                 * recovery_units_factor
-                                * self.hours_per_shift
-                                * self.shifts_per_day
-                                * self.operating_days_per_year
-                                / pyunits.year
+                                # * self.hours_per_shift
```

Comment:
will this stay commented out?

Reply:
Yes, I commented this out when I changed the recovery rate calculations from per-hour to per-year. I will remove these lines.